client/src/components/Header.tsx

```diff
@@ -171,164 +183,189 @@ function Header() {
             <img className={classes.logo} src={Logo} alt="NEH logo placeholder" />
           </NavLink>
           <div>
-            <Button
-              id="exchange-button"
-              aria-label="exchange dropdown"
-              aria-controls={isExchangeMenuOpen ? 'about-menu' : undefined}
-              aria-haspopup="true"
-              aria-expanded={isExchangeMenuOpen ? 'true' : undefined}
-              onClick={handleClick}
-              endIcon={isExchangeMenuOpen ? <ExpandLessIcon /> : <ExpandMoreIcon />}
-              sx={{
-                textTransform: 'capitalize',
-                color: '#323232',
-                marginRight: '10px',
-              }}
-            >
-              Exchange
-            </Button>
-            <Menu
-              id="exchange-menu"
-              anchorEl={exchangeAnchorEl}
-              keepMounted
-              open={isExchangeMenuOpen}
-              onClose={handleClose}
-              MenuListProps={{
-                'aria-labelledby': 'exchange-button',
-              }}
-              anchorOrigin={{
-                vertical: 'bottom',
-                horizontal: 'right',
-              }}
-              transformOrigin={{
-                vertical: 'top',
-                horizontal: 'right',
-              }}
-              PaperProps={{
-                elevation: 0,
-                sx: {
-                  overflow: 'visible',
-                  filter: 'drop-shadow(0px 2px 8px rgba(0,0,0,0.32))',
-                  mt: 1,
-                  marginLeft: '0.7em',
-                  '&:before': {
-                    content: '""',
-                    display: 'block',
-                    position: 'absolute',
-                    top: 0,
-                    right: 20,
-                    width: 10,
-                    height: 10,
-                    bgcolor: 'background.paper',
-                    transform: 'translateY(-50%) rotate(45deg)',
-                    zIndex: 0,
-                    marginLeft: '0.7em',
+            <>
+              <IconButton size="large">{/* Mailbox icon here */}</IconButton>
+              <IconButton size="large">{/* Bell icon here */}</IconButton>
+              <IconButton size="large">{/* Profile icon here */}</IconButton>
+
+              <Button
+                id="exchange-button"
+                aria-label="exchange dropdown"
+                aria-controls={isExchangeMenuOpen ? 'about-menu' : undefined}
+                aria-haspopup="true"
+                aria-expanded={isExchangeMenuOpen ? 'true' : undefined}
+                onClick={handleClick}
+                endIcon={isExchangeMenuOpen ? <ExpandLessIcon /> : <ExpandMoreIcon />}
+                sx={{
+                  textTransform: 'capitalize',
+                  color: '#323232',
```

Comment:
It'll be more maintainable to get the primary/secondary color from the theme definition.

`theme.palette.primary.main`

Theme is imported from `src/theme/index.tsx`